api/integrations/fifabot/.gitignore

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
I would rather not have this generated gitignore here. Just create one with the files you actually need to ignore, not an giant set of files we will never use inside the repo.

Reply:
ok. i'll reset the gitignore.